src/CatalogPage.js

```diff
@@ -1,10 +1,53 @@
+import { it } from "node:test";
+
 /**
  * This class obtains the elements in the DOM for the
  * catalog page using different locators
  */
 class CatalogPage {
   constructor(page) {
     this.productTitle = page.getByTestId("title");
+    this.productPrices = page.getByTestId("inventory-item-price");
+    this.dropdown = page.locator("//select[@data-test='product-sort-container']");
   }
+
+  async getPriceListArray(){
+    const priceItemsArray = [];
+    for(let item of await this.productPrices.all()){
+      const price  = await item.innerText();
+      const priceFloat = parseFloat(price.replace("$", ""));
+      priceItemsArray.push(priceFloat);
+    }
+    return priceItemsArray;
+  }
+
+  async orderArrayAscending(itemsPriceArray){
```

Comment:
This method is a good candidate to be in a `utils.js` file by itself instead of being part of a Page Object Model.

It is a method you could use anywhere in your Automation solution whenever you need to sort an array.

Additionally, you could change the implementation to handle both ascending and descending sorting.

Something like:
```
 /**
   * Sorts the given array.
   *
   * @param {string} order - Valid options: "ascending" or "descending".
   *
   * @returns - Sorted array.
   */
async sortArray(order) {
   ... // Implementation goes here
}
```